app/components/Stepper.tsx

```diff
@@ -0,0 +1,59 @@
+import { Link, useLocation } from '@remix-run/react'
+import React from 'react'
+
+const steps = [
+  { label: 'Add your resume' },
+  { label: 'Add job offer' },
+  { label: 'Enhance resume' },
+]
+
+export default function Stepper() {
+  const location = useLocation()
+  const currentStep = location.pathname.split('/').pop() // e.g. "step1", "step2"
+
```

Comment:
**suggestion**: Consider using a more robust method to determine the current step, such as mapping step names to routes, to avoid potential issues with URL changes.